packages/admin-ui/src/Checkbox/CheckboxGroupPresenter.ts

```diff
@@ -0,0 +1,74 @@
+import { makeAutoObservable } from "mobx";
+import { CheckboxGroupVm } from "./CheckboxGroup";
+import { CheckboxItem } from "./CheckboxItem";
+import { CheckboxItemMapper } from "./CheckboxItemMapper";
+import { CheckboxItemDto } from "./CheckboxItemDto";
+
+interface CheckboxGroupPresenterParams<TValue = string | number> {
+    items: CheckboxItemDto[];
```

Comment:
I was watching the FTA videos a bit, and saw that DTO was being mentioned within the gateway layer. Maybe I need to watch a bit more videos but... it seems that DTO can be used anywhere?

Reply:
We can discuss better naming here as well, but generally, we can use a "DTO" structure for communication between different layers of the architecture.

Reply:
Sounds good. Tnx!